src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/servicebus/PaymentMessageProcessor.java

```diff
@@ -161,7 +161,8 @@ private void deadLetterTheMessage(
             description
         );
 
-        log.info("Message with ID {} has been dead-lettered, reason {} ", message.getMessageId(), reason);
+        log.info("Message with ID {} has been dead-lettered, reason {}, description {}",
+                 message.getMessageId(), reason, description);
```

Comment:
each argument should be in the new line ;)

Reply:
fixed